### PR TITLE
Assignment 1st update

### PR DESCRIPTION
**Pull request by ymmy1**

### CODE REVIEW

File: lesson01/Assignment 02 - Oleg/index.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
+<html lang="en">
```
Comment (ClovisI) [inline lesson01/Assignment 02 - Oleg/index.html:2]: Ensure that you add `<!DOCTYPE html>` as the first line to make this 100% semantically correct.

File: lesson01/Assignment 02 - Oleg/index.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+        <main class="main">
+            <h1>Using ES6+ and jQuery Methods</h1>
+            <section class="input-set">
+                <!-- Input -->
+                <input type="text" class="input">
+                <!-- Submit for Input -->
+                <input class="submit" type="submit"></input>
```
Comment (ClovisI) [inline lesson01/Assignment 02 - Oleg/index.html]: Delete the stray `</input>` tag on line 24.

File: lesson01/Assignment 02 - Oleg/index.html [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+                <!-- Result markup with hook for answer -->
+                <p class="markup">My total with 10% is: <span class="sum"></span></p>
+            </section>
+            <section class="input-set">
+                <!-- Input -->
+                <input type="text" class="input_2">
+                <!-- Submit for Input -->
+                <input class="submit_2" type="submit"></input>
```
Comment (ClovisI) [inline lesson01/Assignment 02 - Oleg/index.html]: Delete the stray `</input>` tag on line 32.

File: lesson01/Assignment 02 - Oleg/index.html [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+                <!-- Result markup with hook for answer -->
+                <p class="markup">My total with 20% is: <span class="sum_2"></span></p>
+            </section>
+            <section class="input-set">
+                <!-- Input -->
+                <input type="text" class="input_3">
+                <!-- Submit for Input -->
+                <input class="submit_3" type="submit"></input>
```
Comment (ClovisI) [inline lesson01/Assignment 02 - Oleg/index.html]: Delete the stray `</input>` tag on line 40.

File: lesson01/Assignment 02 - Oleg/index.html [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+                <!-- Result markup with hook for answer -->
+                <p class="markup">My total with 30% is: <span class="sum_3"></span></p>
+            </section>
+            <section class="input-set">
+                <!-- Input -->
+                <input type="text" class="input_4">
+                <!-- Submit for Input -->
+                <input class="submit_4" type="submit"></input>
```
Comment (ClovisI) [inline lesson01/Assignment 02 - Oleg/index.html]: Delete stray `</input>` tag on line 48.

**Comment by ymmy1** [comment]
Thanks, you are totally right. The reason why nobody ( I suppose 90% of the class ) doesn’t do it. Because we only edit your index.html file. And you don’t add this doctype, and also we are only focused on the task, and wouldn’t even think that did something wrong.


Sent from Mail for Windows 10

**Pull request closed without merging**